openqa-label-known-issues

```diff
@@ -144,7 +144,8 @@ investigate_issue() {
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
         handle_unreachable_or_no_log "$id" "$testurl" "$out"
-    elif label_on_issues_from_issue_tracker "$id"; then
+    fi
+    if label_on_issues_from_issue_tracker "$id"; then
```

Comment:
There is already handling in `handle_unreachable_or_no_log` that will additionally check if the job itself is deleted or not, so I think we shouldn't unconditionally go on here.

I don't have a good suggestion right now for this though.

Reply:
But I didnt change anything in this logic. it is as it was before. And sure to take a look, but it doesnt sound direct connected to the problem in the issue's description. 

I will quote this comment in the ticket

Reply:
tried to improve on https://github.com/os-autoinst/scripts/commit/271c4247ed9b7df701a1cab3f354013639799b58

Reply:
Following the discussion and the comments, new commit 9db6f7d

and dry_run executions 
```
    ~/Documents/Work/qatools/repos/scripts on    164296_allow_known_issues *1 !1                                                                                                                                                                           at   07:32:42
❯ dry_run=1 host=openqa.suse.de ./openqa-label-known-issues https://openqa.suse.de/tests/14897579
Requesting jobs/14897579 via openqa-cli
openqa-cli api --header User-Agent: openqa-label-known-issues (https://github.com/os-autoinst/scripts) --host https://openqa.suse.de --retries=3 -X POST jobs/14897579/comments text=poo#73375 Job incompletes with reason auto_review:"(?m)api failure$" (and no further details)
    ~/Documents/Work/qatools/repos/scripts on    164296_allow_known_issues *1 !1                                                                                                                                                                           at   07:39:42
❯ dry_run=1 host=openqa.suse.de ./openqa-label-known-issues https://openqa.suse.de/tests/15089626
Requesting jobs/15089626 via openqa-cli
[https://openqa.suse.de/tests/15089626](https://openqa.suse.de/tests/15089626): Unknown test issue, to be reviewed
-> [autoinst-log.txt](https://openqa.suse.de/tests/15089626/file/autoinst-log.txt)

Last lines before SUT shutdown:

    # --- 8< ---
    # [2024-08-05T12:10:26.164521+02:00] [debug] [pid:126110] QEMU status is not 'shutdown', it is 'running'
    # [2024-08-05T12:10:26.164636+02:00] [debug] [pid:125698] backend shutdown state: 
    # [2024-08-05T12:10:26.164988+02:00] [info] [pid:126110] ::: OpenQA::Qemu::Proc::save_state: Saving QEMU state to qemu_state.json
    # [2024-08-05T12:10:27.220847+02:00] [debug] [pid:126110] Passing remaining frames to the video encoder
    # frame=  771 fps=1.9 q=0.0 Lsize=    1531kB time=00:00:32.08 bitrate= 390.9kbits/s speed=0.0777x    
    # video:1525kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.355806%
    # [2024-08-05T12:10:27.961098+02:00] [debug] [pid:126110] Waiting for video encoder to finalize the video
    # [2024-08-05T12:10:27.961202+02:00] [debug] [pid:126110] The built-in video encoder (pid 126494) terminated
    # [2024-08-05T12:10:27.961265+02:00] [debug] [pid:126110] The external video encoder (pid 126492) terminated
    # [2024-08-05T12:10:27.961832+02:00] [debug] [pid:126110] QEMU: qemu-system-x86_64: terminating on signal 15 from pid 126110 (/usr/bin/isotovideo: backen)
    # --- >8 ---
    ~/Documents/Work/qatools/repos/scripts on    164296_allow_known_issues *1 !1                                                                                                                                                                           at   07:39:56
❯ dry_run=1 host=openqa.suse.de ./openqa-label-known-issues https://openqa.suse.de/tests/14890572
Requesting jobs/14890572 via openqa-cli
'https://openqa.suse.de/tests/14890572' does not have autoinst-log.txt but is rather old, ignoring
```